src/components/localization/localized-link.js

```diff
@@ -139,7 +139,9 @@ export const LocalizedLink = React.forwardRef(({ to, ...props }, ref) => {
                     target={target}
                     rel={rel}
                     className={className}
-                    style={style}
+                    style={
+                        style ? style : { color: `var(--color-black-3)`, textDecoration: `none` }
```

Comment:
If the value of `style` can have one of these values an `object` or `undefined` we can replace this syntax with `style={style ?? {color:...}}`

Reply:
nice one. didn't know about this operator. done.